README.md

```diff
@@ -65,7 +65,7 @@ OpenLoco requires original files of Chris Sawyer's Locomotion to play. It can be
 - cmake
 - make or ninja
 - [SDL2](https://www.libsdl.org/download-2.0.php)
-- [yaml-cpp](https://github.com/jbeder/yaml-cpp)
+- [yaml-cpp](https://github.com/jbeder/yaml-cpp) (lib32-yaml-cpp on arch)
```

Comment:
I don't think we should be making distro-specific package notes here. All these packages should be installed in their 32-bit variants, not just yaml-cpp...